Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1

```diff
@@ -156,6 +156,14 @@ function Get-TargetResource
     $ConnectionModeTasks = New-M365DSCConnection -Workload 'Tasks' `
         -InboundParameters $PSBoundParameters
 
+    # Workaround for issue when if connected to S+C prior to calling cmdlet, an error about an invalid token is thrown.
```

Comment:
Shouldn't we add this within MSCloudLoginAssistant or is this just the case for this resource?

Reply:
Just this resource.

Reply:
A bug was raised with the EXO Management Shell team about it.

Reply:
ok, then!